help/pt_PT/intro/topic/000001.xml

```diff
@@ -53,7 +53,7 @@
 	</section>
 	<section title='4. Por que aparece a mensagem "Esta revista não aceita submissões neste momento"?'>
 		<![CDATA[
-<p>Caso o Editor-Gestor ainda não tenha criado nenhuma secção para a revista (por exemplo, Artigos, Livros, Revisões, etc.), os autores verão esta mensagem durante o processo de submissão. Também, caso o Editor-Gestor tenha activado a opção "Restrita a Editores e Editores de Secção", para todas as secções criadas, somente os Editores e Editores de Secção poderão submeter à revista.</p>
```

Comment:
This is probably a subject different translators will fight over, but as long as we're consistent, I don't mind merging this. But let me spot-review it a little more first.